network/address_test.go

```diff
@@ -27,6 +29,12 @@ func TestConnType(t *testing.T) {
 }
 
 func TestAddress(t *testing.T) {
+	localhostNetworkAddresses, _ := net.LookupHost("localhost")
+	sort.Strings(localhostNetworkAddresses)
+	facebookNetworkAddresses, _ := net.LookupHost("facebook.com")
+	sort.Strings(facebookNetworkAddresses)
+	googleNetworkAddresses, _ := net.LookupHost("google.com")
```

Comment:
+1 for @ineiti 's suggestion. You don't need to change any method's signature. Just change `net.LookupHost(host)` in `NetworkAddress()` func, by `lookupHost(host)`, as defined by @ineiti just above. The other functions don't need to know about that.